modules/tripleliftBidAdapter.js

```diff
@@ -241,6 +241,9 @@ function _getORTBVideo(bidRequest) {
     if (!video.placement) {
       video.placement = 1;
     }
+    if (!video.plcmt) {
+      video.plcmt = 1;
```

Comment:
@fowler446 can you comment here? Generally video.context instream does not map to plcmt = 1

Reply:
![image (8)](https://github.com/prebid/Prebid.js/assets/20526608/5963b311-2ef9-4771-ba36-dce49b7f844e)

Reply:
This is direct from IAB so I think we have instream `plcmt` correct. But like I mentioned below, if the publisher sets this, we don't meddle with it.

Reply:
You're gonna get in trouble here from ttd. They were very adamant no one does this. Prebid instream context basically just means preroll, there is no easy way for you to infer if plcmt should be 1 or 2